eng/restore/netcoreapp/netcoreapp.depproj

```diff
@@ -4,15 +4,21 @@
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.7</NETCoreAppPackageVersion>
-    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>
+    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>        
+    <NETCoreAppRefPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp3.0'">3.0.0-preview7-27912-14</NETCoreAppRefPackageVersion>
```

Comment:
FYI - preview 8 just shipped yesterday.

BTW - what is the plan for keeping these versions up to date? Do we just do it manually/ad hoc?